application/src/main/data/upgrade/3.6.2/schema_update_attribute_kv.sql

```diff
@@ -0,0 +1,181 @@
+    IF EXISTS(SELECT 1 FROM information_schema.tables WHERE table_name = 'attribute_kv_old') THEN
+        EXECUTE format('COPY (SELECT records.entity_id                           AS entity_id,
+                               to_attribute_type_id(records.attribute_type)  AS attribute_type,
+                               records.attribute_key                         AS attribute_key,
+                               records.bool_v                                AS bool_v,
+                               records.str_v                                 AS str_v,
+                               records.long_v                                AS long_v,
+                               records.dbl_v                                 AS dbl_v,
+                               records.json_v                                AS json_v,
+                               records.last_update_ts                        AS last_update_ts
+                   FROM (SELECT entity_id,
+                               attribute_type,
+                               key_id AS attribute_key,
+                               bool_v,
+                               str_v,
+                               long_v,
+                               dbl_v,
+                               json_v,
+                               last_update_ts
+                        FROM attribute_kv_old INNER JOIN attribute_kv_dictionary ON (attribute_kv_old.attribute_key = attribute_kv_dictionary.key)
+                        WHERE attribute_type= ANY(%L)) AS records) TO %L;', attribute_scope_array, path_to_file);
+        EXECUTE format('COPY attribute_kv FROM %L', path_to_file);
+        SELECT COUNT(*) INTO row_num_old FROM attribute_kv_old;
+        SELECT COUNT(*) INTO row_num FROM attribute_kv;
+        RAISE NOTICE 'Migrated % of % rows', row_num, row_num_old;
+    END IF;
+EXCEPTION
+    WHEN others THEN
+        ROLLBACK;
+        RAISE EXCEPTION 'Error during COPY: %', SQLERRM;
+END
+$$;
+
+CREATE OR REPLACE PROCEDURE recreate_device_info_active_attribute_view()
+    LANGUAGE plpgsql AS
+$$
+BEGIN
+    DROP VIEW IF EXISTS device_info_active_attribute_view CASCADE;
+    CREATE OR REPLACE VIEW device_info_active_attribute_view AS
+    SELECT d.*
+         , c.title as customer_title
+         , COALESCE((c.additional_info::json->>'isPublic')::bool, FALSE) as customer_is_public
+         , d.type as device_profile_name
+         , COALESCE(da.bool_v, FALSE) as active
+    FROM device d
+             LEFT JOIN customer c ON c.id = d.customer_id
+             LEFT JOIN attribute_kv da ON da.entity_id = d.id AND da.attribute_type = 2 AND da.attribute_key = (select key_id from attribute_kv_dictionary where key = 'active');
+END;
+$$;
+
+CREATE OR REPLACE PROCEDURE recreate_device_info_view()
+    LANGUAGE plpgsql AS
+$$
+BEGIN
+    DROP VIEW IF EXISTS device_info_view CASCADE;
+    CREATE OR REPLACE VIEW device_info_view AS SELECT * FROM device_info_active_attribute_view;
+END;
+$$;
```

Comment:
I think we need to just drop the view, as it will be created with createOrUpdateViewsAndFunctions